src/main/java/bio/terra/cli/serialization/persisted/PDWorkspace.java

```diff
@@ -28,7 +29,8 @@ public class PDWorkspace {
 
   /** Serialize an instance of the internal class to the disk format. */
   public PDWorkspace(Workspace internalObj) {
-    this.id = internalObj.getId();
+    this.uuid = internalObj.getUuid();
```

Comment:
Are you attempting backward compatibility? This will break that. I'd be inclined to keep the `id` since it's a JSON field.

Reply:
Maybe okay to break compatibility now.  But... maybe we want to store UUID on the disk as the "current workspace"?